app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -86,10 +91,47 @@ class AdministratorControlsActivityPresenter @Inject constructor(
     getAdministratorControlsFragment()!!.setSelectedFragment(lastLoadedFragment)
     activity.supportFragmentManager.beginTransaction().add(
       R.id.administrator_controls_fragment_multipane_placeholder,
-      AppVersionFragment()
+      AppVersionFragment(),
+      ADMINISTRATOR_CONTROLS_BACKSTACK
     ).commitNow()
   }
 
+  fun loadProfileEdit(profileId: Int) {
+    lastLoadedFragment = PROFILE_EDIT_FRAGMENT
+    binding.administratorControlsMultipaneOptionsBackButton!!.visibility = View.VISIBLE
+    val fragment = ProfileEditFragment.newInstance(profileId, isMultipane)
+    activity.supportFragmentManager.beginTransaction().add(
+      R.id.administrator_controls_fragment_multipane_placeholder,
+      fragment,
+      ADMINISTRATOR_CONTROLS_BACKSTACK
+    )
+      .addToBackStack(ADMINISTRATOR_CONTROLS_BACKSTACK)
+      .commit()
+  }
+
+  fun handleOnBackPressed() {
+    if (activity.supportFragmentManager.backStackEntryCount > 0)
+      activity.supportFragmentManager.popBackStackImmediate()
+  }
+
+  fun handleOnResume() {
+    activity.supportFragmentManager.addOnBackStackChangedListener {
+      if (activity.supportFragmentManager.backStackEntryCount == 0)
+        binding.administratorControlsMultipaneOptionsBackButton!!.visibility = View.GONE
+      val multipane_id = R.id.administrator_controls_fragment_multipane_placeholder
+      val multipane_fragment =
+        activity.supportFragmentManager.findFragmentById(multipane_id)
+      if (multipane_fragment is ProfileListFragment) {
+        setExtraControlsTitle("Edit profiles")
```

Comment:
This string should be defined in strings.xml file

Reply:
done